helm/hpcc/templates/localroxie.yaml

```diff
@@ -121,21 +121,28 @@ spec:
 {{ include "hpcc.addUDPCertificateVolume" (dict "root" $ "name" $roxie.name "component" "localudpkey" ) | indent 6 }}
 ---
 {{- range $service := $roxie.services }}
-{{- if ne (int $service.port)  0 }}
+{{- if ne (int $service.servicePort)  0 }}
 apiVersion: v1
 kind: Service
 metadata:
   name: {{ $service.name | quote }}
   labels:
     type: roxie-service
+{{- if hasKey $service "labels" }}
+{{ toYaml $service.labels | indent 4 }}
+{{ end -}}
+{{- if hasKey $service "annotations" }}
+  annotations:
+{{ toYaml $service.annotations | indent 4 }}
+{{ end -}}
 spec:
   ports:
-  - port: {{ $service.port }}
+  - port: {{ $service.servicePort }}
     protocol: TCP
-    targetPort: {{ $service.port }}
+    targetPort: {{ $service.servicePort }}
```

Comment:
shouldn't this remain as port?

Reply:
Roxie does not allow them to be different. I'm not sure why other components do, TBH.